app/controllers/solidus_pay_tomorrow/base_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module SolidusPayTomorrow
+  class BaseController < ::Spree::Api::BaseController
```

Comment:
If I don't use `skip_before_action` hooks defined in `::Spree::Api::BaseController`, I get - 
`You must specify an API key` hence I have inherited from  `::Spree::Api::BaseController` after taking reference from [Bolt](https://github.com/solidusio/solidus_bolt/blob/master/app/controllers/solidus_bolt/base_controller.rb)

Reply:
Yes, but here the purpose is different. In Bolt we skipped the authentication because the call came from their services, and they can't be authorized on our side with the bearer authentication. Still, as you can see, there is a [verify_bolt_request](https://github.com/solidusio/solidus_bolt/blob/master/app/controllers/solidus_bolt/base_controller.rb#L13-L19) method responsible for verifying that the request is made by Bolt.
The API here is used by our FE, and we should use the guest token to authorize it.

Reply:
Added guest_token in https://github.com/nebulab/solidus_pay_tomorrow/pull/9/commits/7af9fc62fba5a622a479931f394c675b2183f422